### ISSUE TITLE
offline versions

### ISSUE THREAD

**Issue by elburz** [openframeworks/ofBook#232]
Is it possible to periodically build the pdf version of the ofBook and have a dropbox link for it somewhere near the beginning in the book? I think there are those that would use the offline version but seems a bit difficult to have to get the whole repo up and running to create your own pdf verison.

**Comment by golanlevin** [comment]
I agree that it would be great to have a more recent build of the PDF. The most recent version I can find is from 2015, and I know a lot has been done since then.

**Comment by ofZach** [comment]
At some point, in order to get the book published, we made the decision collectively to scrap the PDF book export (which had some bugs) and focus completely on the things we needed to get done for the web version.  I'm not 100% what the status of the PDF export is but just wanted to mention this was a conscious decision to focus our energies on that....   at any rate, I will investigate the PDF export, I remember there were issues previously...

**Comment by elburz** [comment]
@golanlevin where did you find the pdf? I have a tough time finding it

@ofZach understandable. Maybe something for further down the line, or possibly now if the development has slowed down due to time constraints, but we recently moved our similar project to gitbooks platform. Not perfect platform by any means but seems to be working for us with a similar folder/markdown layout as this project, and the nice thing is that it automatically builds web version, epub, mobi, and pdf, directly from the repo commits. We had converted to this from latex since we found people weren't so keen on digging into latex. The conversion was a bit of a pain, but if everything here is already markdown, the process shouldn't be too bad. If it seems like a good decision, I could see about allocating some time to that. 

https://www.gitbook.com/book/nvoid/introduction-to-touchdesigner/details

**Comment by mikewesthad** [comment]
Here's a fresh build: https://github.com/openframeworks/ofBook/releases/tag/v1.0.0.

After we made that decision to scrap the PDF book, I occasionally made some small tweaks to keep that pipeline limping along. It's currently broken by the SVGs in the memory chapter (since vanilla latex doesn't support SVG).  The old hack that was in place to fix that doesn't work, and I don't have the bandwidth right now to look into it.

**Comment by elburz** [comment]
No problem, thanks for the effort in getting the fresh build up.
